Tasks/GooglePlayRelease/typings/modules/glob/typings.json

```diff
@@ -1,17 +0,0 @@
-{
-  "resolution": "main",
```

Comment:
Since typings have been removed - do we still need 'typings' dependency in package.json?